### DISCUSSION TITLE
Example like Needle.tools home page?

### DISCUSSION THREAD

**Discussion by JohnDietrich-Pepper** [needle-tools/needle-engine-support#132] [Demo and Starter Ideas]
### Describe the bug 💬

Incredibly useful project.  Any chance that we could get an example like the Needle.tools home page? Having trouble duplicating the scroll from 3d engine to 3d engine and would like to understand how you're overlaying the text/menu.

### Operating System 👩‍💻

Windows, macOS, Linux, iOS (mobile), Android (mobile)

### What browsers are you seeing the problem on? 🏄‍♂️

Chrome, Microsoft Edge

### Editor Version 🎲

2021.3

### Needle Exporter version

2.4 pre

### Steps to reproduce 🔢

```bash
1. Go to needle.tools
2.
3.
...
```

### Validations 🩹

- [X] Follow our [Code of Conduct](https://github.com/needle-tools/needle-engine-support/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://docs.needle.tools).
- [X] Check that there isn't [already an issue](https://github.com/needle-tools/needle-engine-support/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Needle-Engine-specific issue. For example if it's a Unity related bug, it should be reported to [Unity](https://unity3d.com/unity/qa/bug-reporting) instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/needle-tools/needle-engine-support/discussions) or join our [Discord Chat Server](https://discord.needle.tools).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by marwie** [comment]
Hello @JohnDietrich-Pepper 

have you had a look at the scrollytelling sample we provide [here](https://engine.needle.tools/samples/?room=wise-skyblue-monkey&open=1#product-scrollytelling)? It's kind of similar to how it's done on our website (on the website we just split the content into multiple scenes that are loaded in and out at specific points)

We also used the CSS tracks that we have as custom timeline tracks (you can get those scripts [here](https://github.com/needle-tools/needle-engine-modules))

Hope this helps you

**Comment by JohnDietrich-Pepper** [comment]
Thanks that is very helpful I had overlooked that one.

One more question say you have an object the user clicks.  I would like to then change the contents of a div on the page.  It seems like this shouldn't be incredibly difficult but is this the recommended approach or would it be better to edit the text of an UI object in the engine?

**Comment by marwie** [comment]
Here is a small code sample: https://engine.needle.tools/docs/scripting-examples.html#Receive%20Clicks%20on%20Objects
It depends on your usecase and scene. Both can work well :)

**Comment by JohnDietrich-Pepper** [comment]
Pretty awesome.  Was able to get it working.  Would be very helpful to get more script references like switching out shaders and handling post processing like OnRenderImage on the camera.

**Comment by JohnDietrich-Pepper** [comment]
Does subscription come with the bike store and needle tools home page examples?

**Comment by marwie** [comment]
It comes with the bike, castle and monster hands repo right now (we'll add a sample of our website later)
